typings/jsonld.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'jsonld' {
+  export function canonize(doc: any): string
+  export function expand(doc: any): any
```

Comment:
Just to make sure I'm not missing anything — is `expand` not being used anywhere? (Nothing wrong with that, just asking)